projects/netgrif-components-core/src/lib/view/task-view/abstract-single-task-view.component.ts

```diff
@@ -48,6 +59,9 @@ export abstract class AbstractSingleTaskViewComponent extends AbstractViewWithHe
         if (!!this.subPanelData) {
             this.subPanelData.unsubscribe();
         }
+        if (!!this.taskPanelData) {
+            this.taskPanelData.unsubscribe();
```

Comment:
we don't want to `unsubscribe` a subject, we can call `close` if we want to dispose of it

Reply:
I think you meant to do `complete`. We either `unsubscribe` a subscription or `complete` a subject/observable, there is no "close" method on that subject. It suggests the unsubscribe method too, but I agree, that in that case complete would be better.

Reply:
yes, I meant `complete`. Thank you for correcting me